components/Quiz.tsx

```diff
@@ -0,0 +1,95 @@
+"use client"
+
+import { Button } from "@shadcn/ui/button"
+import { Checkbox } from "@shadcn/ui/checkbox"
+import { Label } from "@shadcn/ui/label"
+import { RadioGroup, RadioGroupItem } from "@shadcn/ui/radio-group"
+
+import { Typography } from "./Typography"
+
+type Answer = {
+  text: string
+  isCorrect: boolean
+}
+
+interface Props {
+  question: string
+  answers: Answer[]
+}
+
+export function Quiz({ question, answers }: Props) {
+  const isMultipleCorrect = answers.filter((answer) => answer.isCorrect).length > 1
+
+  async function onSubmit(formdata: FormData) {
+    const selectedAnswers = Array.from(formdata.getAll("answer"))
+
+    /**
+     * Return if no value is selected
+     */
+    if (selectedAnswers.length < 1) return
```

Comment:
**suggestion:** we shouldn't allow Submit when nothing is selected. Then we won't need this if.

Reply:
Okay.
We should get the status from HTML. Right?

Reply:
Okay, HTML takes care of this for us.